package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ssh-socket-action",
-  "version": "1.1.2",
+  "version": "1.4.0",
```

Comment:
Does the lock file need updating too?

Reply:
I believe npm would, but I don't see any file changes on yarn1.